src/commands.ts

```diff
@@ -525,3 +525,31 @@ export const openCmd = async (scriptId: any) => {
     process.exit(0);
   }
 };
+
+/**
+ * Acts as a router to apis subcommands
+ * Calls functions for list, enable, or disable
+ * Otherwise returns an error of command not supported
+ */
+export const apis = async () => {
+  const list = async () => {
+    await checkIfOnline();
+    const {data} = await discovery.apis.list({
+      preferred: true,
+    });
+    data.items.forEach((api) => {
+      console.log(`${api.name.padEnd(25)} - ${api.id.padEnd(30)}`);
+    });
+  };
+  const subcommand: string = process.argv[3]; // clasp apis list => "list"
+  const command: any = {
```

Comment:
No `any`. Use a map.
I think it's like this:
`var x: {[key: KeyType]: ValueType} = {}`

Reply:
Or maybe `Map<string, string>`